e2e/updatecli.d/success.d/golang/gomod.yaml

```diff
@@ -0,0 +1,37 @@
+name: "Test gomod"
+
+sources:
+  default:
+    name: Get version
+    kind: golang/gomod
+    spec:
+      file: pkg/plugins/resources/go/gomod/testdata/go.mod
+      module: "github.com/Masterminds/semver/v3"
+
+  golang:
+    name: Get version
+    kind: golang/gomod
+    spec:
+      file: pkg/plugins/resources/go/gomod/testdata/go.mod
+      kind: golang
```

Comment:
Is this `kind` needed since it's already a `golang/mod` kind?

Reply:
This kind is used to define if we want to retrieve the Golang version or a module version but I can remove it. 
If a module is specified we assume we want to retrieve a module version otherwise we retrieve the golang  version specified in the go.mod file